### ISSUE TITLE
Tests aren't passed

### ISSUE THREAD

**Issue by zdm** [xslate/p5-Text-Xslate#154]
Env:
strawberry perl 5.22.1.1 x64
Text::Xslate v3.3.8

```
t/900_bugs/021_cached_enc.t ..................... ok
t/900_bugs/022_empty_if_block.t ................. ok
t/900_bugs/023_deploy_problem.t ................. IO::Handle::sync not implemented on this architecture at t/900_bugs/023_deploy_problem.t line 54.
t/900_bugs/023_deploy_problem.t ................. Dubious, test returned 255 (wstat 65280, 0xff00)
No subtests run
t/900_bugs/024_use_cache.t ...................... ok
t/900_bugs/025_clobber-macro-args.t ............. ok
t/900_bugs/026_issue61.t ........................ ok
t/900_bugs/027_issue65.t ........................ ok
t/900_bugs/028_issue68.t ........................ ok
t/900_bugs/029_fork_and_cache.t ................. skipped: deal with memorycache-filecache-original model more effectively
```

**Comment by syohex** [comment]
@zdm Could you test with this patch(https://github.com/xslate/p5-Text-Xslate/pull/155.diff) ?

**Comment by zdm** [comment]
Tested, worked.

**Comment by syohex** [comment]
Thanks for checking.
I have released 3.3.9. Please check it.

https://metacpan.org/release/SYOHEX/Text-Xslate-3.3.9

**Comment by zdm** [comment]
Installed.
Thank you.
